### PR TITLE
Add investigation guides

### PR DESCRIPTION
**Pull request by shashank-elastic**
## Issues
#2325 

## Summary
Add Triage and Analysis to rules

1. rules/linux/credential_access_potential_linux_ssh_bruteforce_root.toml
2. rules/linux/credential_access_potential_linux_ssh_bruteforce.toml
3. rules/linux/credential_access_bruteforce_passowrd_guessing.toml

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

**Review by w0rk3r — CHANGES_REQUESTED** [review]
Hey, you can check the proper format that investigation guides must follow on [other security content PRs](https://github.com/elastic/detection-rules/pulls?q=is%3Apr+label%3A%22Security+Content%22+). Let me know if you have questions

**Comment by shashank-elastic** [comment]
> Hey, you can check the proper format that investigation guides must follow on [other security content PRs](https://github.com/elastic/detection-rules/pulls?q=is%3Apr+label%3A%22Security+Content%22+). Let me know if you have questions

@w0rk3r I could see Spacing was an issue from the PR reference you have quoted and I have corrected the same. The guide is just a Linux adaptation of what is already merged into the main for Windows. Please check https://github.com/elastic/detection-rules/pull/2275/files

File: rules/linux/credential_access_bruteforce_passowrd_guessing.toml [hunk 1]
```diff
@@ -16,6 +16,37 @@ index = ["auditbeat-*", "logs-system.auth-*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Potential SSH Password Spraying"
+note = """## Triage and analysis
+
```
Comment (terrancedejesus) [inline rules/linux/credential_access_bruteforce_passowrd_guessing.toml:20]: Should we add `### Investigating Potential SSH Password Spraying` as a header here to remain consistent with the flow of other investigation guides?

File: rules/linux/credential_access_potential_linux_ssh_bruteforce.toml [hunk 2]
```diff
@@ -17,6 +17,35 @@ index = ["auditbeat-*", "logs-system.auth-*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Potential SSH Brute Force Detected"
+note = """## Triage and analysis
+
```
Comment (terrancedejesus) [inline rules/linux/credential_access_potential_linux_ssh_bruteforce.toml:21]: Should we add ### Investigating Potential SSH Brute Force Detected as a header here to remain consistent with the flow of other investigation guides?

**Review by Samirbous — APPROVED** [review]

File: rules/linux/credential_access_potential_linux_ssh_bruteforce_root.toml [hunk 3]
```diff
@@ -17,6 +17,33 @@ index = ["auditbeat-*", "logs-system.auth-*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Potential SSH Brute Force Detected on Privileged Account"
+note = """## Triage and analysis
+
```
Comment (terrancedejesus) [inline rules/linux/credential_access_potential_linux_ssh_bruteforce_root.toml:21]: Should we add `### Investigating Potential SSH Brute Force Detected on Privileged Account` as a header here to remain consistent with the flow of other investigation guides?

File: rules/linux/credential_access_potential_linux_ssh_bruteforce_root.toml [hunk 4]
```diff
@@ -17,6 +17,38 @@ index = ["auditbeat-*", "logs-system.auth-*"]
...
+### Investigating Potential SSH Brute Force Attack on Privileged Account
+
+The rule identifies consecutive SSH login failures targetting a privileged(root) account from the same source ip
```
Comment (terrancedejesus) [inline rules/linux/credential_access_potential_linux_ssh_bruteforce_root.toml:24]:
```suggestion
The rule identifies consecutive SSH login failures targetting a privileged (root) account from the same source ip
```

File: rules/linux/credential_access_potential_linux_ssh_bruteforce.toml [hunk 5]
```diff
@@ -17,6 +17,40 @@ index = ["auditbeat-*", "logs-system.auth-*"]
...
+### Investigating Potential SSH Brute Force Attack
+
+The rule identifies consecutive SSH login failures targetting a user account from the same source ip address to the
```
Comment (terrancedejesus) [inline rules/linux/credential_access_potential_linux_ssh_bruteforce.toml:24]:
```suggestion
The rule identifies consecutive SSH login failures targeting a user account from the same source ip address to the
```

File: rules/linux/credential_access_bruteforce_passowrd_guessing.toml [hunk 6]
```diff
@@ -15,7 +15,43 @@ from = "now-9m"
...
 language = "eql"
 license = "Elastic License v2"
-name = "Potential SSH Password Spraying"
+name = "Potential SSH Password Guessing"
+note = """## Triage and analysis
+
+### Investigating Potential SSH Password Guessing Attack
+
+The rule identifies consecutive SSH login failures followed by a successful login from the same source ip address to the
+same target host indicating a successfull attempt of brute force password guessing attack.
```
Comment (terrancedejesus) [inline rules/linux/credential_access_bruteforce_passowrd_guessing.toml:24]:
```suggestion
same target host indicating a successful attempt of brute force password guessing.
```

File: rules/linux/credential_access_bruteforce_passowrd_guessing.toml [hunk 7]
```diff
@@ -15,7 +15,43 @@ from = "now-9m"
 index = ["auditbeat-*", "logs-system.auth-*"]
 language = "eql"
 license = "Elastic License v2"
-name = "Potential SSH Password Spraying"
+name = "Potential SSH Password Guessing"
+note = """## Triage and analysis
+
+### Investigating Potential SSH Password Guessing Attack
+
+The rule identifies consecutive SSH login failures followed by a successful login from the same source ip address to the
```
Comment (terrancedejesus) [inline rules/linux/credential_access_bruteforce_passowrd_guessing.toml:23]:
```suggestion
The rule identifies consecutive SSH login failures followed by a successful login from the same source IP address to the
```

File: rules/linux/credential_access_potential_linux_ssh_bruteforce.toml (hunk 5 above)
Comment (terrancedejesus) [inline rules/linux/credential_access_potential_linux_ssh_bruteforce.toml:24]:
```suggestion
The rule identifies consecutive SSH login failures targetting a user account from the same source IP address to the
```

**Comment by w0rk3r** [comment]
It was more related to the description of the rule, a background of the attacks, giving context to the analysts. But I'll apply my suggestions in the same PR as the windows ones, as you already merged these. 

Also, the point of doing separated PRs for investigation guides is to get reviews from the docs team, which didn't happened here. It would be nice if you wait my review/approval in Investigation Guides related PRs in the future. Thanks.

**Pull request merged**